### PR TITLE
Adding texts in verify_leader metric

### PR DESCRIPTION
**Pull request by 20sr20**
- Added description providing example case when the metric can go high

### CODE REVIEW

File: website/content/docs/agent/telemetry.mdx [hunk 1]
```diff
@@ -398,7 +398,7 @@ These metrics are used to monitor the health of the Consul servers.
...
+| `consul.raft.verify_leader`                         | This metric doesn't have direct correlation to the leader change.  It just counts the number of times an agent checks whether it is still the leader or not.  For example, during every consistent read, the check is done.  So, depending on the load in the system, this metric can be high as it is incremented each time a consistent read is done.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | checks / interval                 | Counter |
```
Comment (karl-cardenas-coding) [inline website/content/docs/agent/telemetry.mdx:401]:
```suggestion
| `consul.raft.verify_leader`                         | This metric doesn't have a direct correlation to the leader change.  It just counts the number of times an agent checks if it is still the leader or not.  For example, during every consistent read, the check is done.  Depending on the load in the system, this metric count can be high as it is incremented each time a consistent read is completed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | checks / interval                 | Counter |
```
Minor language cleanup

**Review by karl-cardenas-coding — APPROVED** [review]
👍🏼

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/545764.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 3f4cf0e64ad548b5b4ff5cbbf1b9f49c5eec8fae onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 3f4cf0e64ad548b5b4ff5cbbf1b9f49c5eec8fae onto `release/1.11.x` succeeded!

**Pull request merged**